util-core/src/test/scala/com/twitter/util/TimeTest.scala

```diff
@@ -307,19 +302,36 @@ trait TimeLikeSpec[T <: TimeLike[T]] extends WordSpec with GeneratorDrivenProper
     }
 
     "deal with undefineds" in {
-      assert(Bottom.floor(1.second) === Bottom)
       assert(Undefined.floor(0.seconds) === Undefined)
       assert(Undefined.floor(Duration.Top) === Undefined)
       assert(Undefined.floor(Duration.Bottom) === Undefined)
       assert(Undefined.floor(Duration.Undefined) === Undefined)
     }
 
-    "floor itself" in {
+    "round to itself" in {
       for (s <- Seq(Long.MinValue, -1, 1, Long.MaxValue); t = fromNanoseconds(s))
         assert(t.floor(Duration.fromNanoseconds(t.inNanoseconds)) === t)
     }
   }
 
+  "floor" should {
+    "round down" in {
+      assert(fromSeconds(60).floor(1.minute) === fromSeconds(60))
```

Comment:
no need for === anymore, scalatest 2.x supplied an assert macro which unpacks failures very neatly even when we use regular old ==.